indico/modules/events/sessions/client/js/session_display.jsx

```diff
@@ -0,0 +1,27 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2021 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {IButton, ICSCalendarLink} from 'indico/react/components';
+import {Translate} from 'indico/react/i18n';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const userId = document.querySelector('body').dataset.userId;
+  const {eventId, sessionId} = document.querySelector('#session-calendar-link').dataset;
```

Comment:
I would assign the element to a variable instead of querying it here and again below. Also, I think this code should be a bit more resilient against the element not existing (just return early if querying it doesn't return something)